switchsdk/src/main/java/net/gini/switchsdk/TakePictureActivity.java

```diff
@@ -63,6 +63,12 @@
             "BUNDLE_EXTRA_BUTTON_ANALYZED_TEXT_SIZE";
     static final String BUNDLE_EXTRA_PREVIEW_FAILED_TEXT = "BUNDLE_EXTRA_PREVIEW_FAILED_TEXT";
     static final String BUNDLE_EXTRA_PREVIEW_SUCCESS_TEXT = "BUNDLE_EXTRA_PREVIEW_SUCCESS_TEXT";
+    /**
+     * Whether the "Analyzing Complete" screen should be skipped (because the customer has its own
```

Comment:
I would remove this comment. It's just the name of a property in the intent.

Reply:
a0d6ccbfe6f283865d2c68e795f2dbdf04553ad9